main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -15,59 +15,89 @@
  */
 package ca.uwaterloo.flix.tools.pkg
 
-import ca.uwaterloo.flix.tools.Packager
 import ca.uwaterloo.flix.tools.Packager.getLibraryDirectory
+import ca.uwaterloo.flix.tools.pkg.Dependency.FlixDependency
 import ca.uwaterloo.flix.tools.pkg.github.GitHub
-import ca.uwaterloo.flix.util.Result.ToOk
-import ca.uwaterloo.flix.util.{Options, Result}
+import ca.uwaterloo.flix.util.Result.{Err, Ok, ToOk}
+import ca.uwaterloo.flix.util.Result
 
-import java.io.File
+import java.io.{IOException, PrintStream}
 import java.nio.file.{Files, Path, StandardCopyOption}
 import scala.util.Using
 
 object FlixPackageManager {
 
   // TODO: Move functionality from "Packager" in here.
 
+  //TODO: tests - how?
```

Comment:
I think we should have a test suite -- which is NOT part of the standard test suite -- that we can run separately which just tries to read a TOML input and download its packages to a temporary directory. See e.g. `Files.createTempDirectory`.